src/pages/Register.jsx

```diff
@@ -1,98 +1,83 @@
-// // Import necessary components, hooks, and stores.
-// import Logos from "../components/Logos";
-// import { userStore } from "../stores/userStore";
-// import { useState } from "react";
-// import { useNavigate } from "react-router-dom";
+
+import { userStore } from '../store/userStore';
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 // import { Link } from "react-router-dom";
+import './Register.css'
 
-// // Define the 'Register' functional component.
 export const Register = () => {
-//   // Initialize state variables for 'username', 'email', and 'password' using 'useState'.
-//   const [username, setUsername] = useState("");
-//   const [email, setEmail] = useState("");
-//   const [password, setPassword] = useState("");
-//   // Initialize the 'navigate' function from React Router.
-//   const navigate = useNavigate();
 
-//   // Access the 'handleSignup' function from the 'userStore'.
-//   const storeHandleSignup = userStore((state) => state.handleSignup);
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+  const storeHandleRegister = userStore(state => state.handleRegister)
+
+  const onRegisterClick = async () => {
+    if (!username || !email || !password) {
+      alert('Please enter username, email, and password');
+      return
+    }
+    try {
+      await storeHandleRegister(username, email, password)
+      //console.log("storeHandleRegister ", storeHandleRegister);
+      console.log("username ", username);
 
-//   // Function to handle the click event of the signup button.
-//   const onSignupClick = async () => {
-//     if (!username || !password || !email) {
-//       // Display an alert if any of the required fields are empty.
-//       alert("Please enter email, username, and password");
-//       return;
-//     }
-//     try {
-//       // Call the 'handleSignup' function from 'userStore' with 'username', 'password', and 'email' parameters.
-//       await storeHandleSignup(username, password, email);
-//       if (username && password) {
-//         // If the signup is successful, navigate to the login route ("/").
-//         navigate("/"); // Replace with your desired path
-//       }
-//     } catch (error) {
-//       // Handle any errors that occur during signup and display an alert.
-//       console.error("Signup error:", error);
-//       alert("An error occurred during signup");
-//     }
-//   };
+      if (username && password) {
+        const appContainer = document.getElementById("app-container")
+        appContainer.innerText = `Registration successful`
+        navigate('/')
+      }
```

Comment:
This is great! Should we also change the global state "isLoggedIn"? Or does that happen automatically